components/logs-table/index.tsx

```
@@ -9,38 +9,63 @@ import { genericError } from "@/lib/errors"
   const [pageSize, setPageSize] = useState(() => {
-    // Initialize from localStorage if available, otherwise use default
-    if (typeof window !== "undefined") {
-      const stored = localStorage.getItem(PAGE_SIZE_STORAGE_KEY)
-      return stored && pageSizeOptions.some((option) => option.value === Number(stored))
-        ? Number(stored)
-        : DEFAULT_PAGE_SIZE
-    }
-    return DEFAULT_PAGE_SIZE
+    if (typeof window === "undefined") return DEFAULT_PAGE_SIZE
+    const stored = localStorage.getItem(PAGE_SIZE_STORAGE_KEY)
+    return stored ? parseInt(stored, 10) : DEFAULT_PAGE_SIZE
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)